packages/core/src/Modules/Endpoint/__tests__/FeatureFlagsSection.spec.tsx

```diff
@@ -56,19 +54,26 @@ describe('<FeatureFlagsSection/>', () => {
         render(mockFeatureFlagsSection());
 
         expect(screen.getByText(feature_flags_title)).toBeInTheDocument();
-        expect(screen.getByRole('checkbox', { name: FLAGS.NEXT_WALLET })).not.toBeChecked();
         expect(screen.getByRole('checkbox', { name: FLAGS.SHARKFIN })).not.toBeChecked();
         expect(screen.getByRole('checkbox', { name: FLAGS.DTRADER_V2 })).not.toBeChecked();
         expect(screen.queryByRole('checkbox', { name: FLAGS.WALLET })).not.toBeInTheDocument();
     });
-    it('should render checked next_wallet, sharkfin & dtrader_v2 flags on localhost', () => {
+    it('should call feature_flags.update() method when a flag is checked', () => {
+        const update = jest.fn();
+        default_mock_store.feature_flags.update = update;
+        render(mockFeatureFlagsSection());
+
+        userEvent.click(screen.getByRole('checkbox', { name: FLAGS.DTRADER_V2 }));
+
+        expect(update).toBeCalled();
+    });
```

Comment:
Have moved this test case to be the first one, because weirdly it doesn't pass in PR CI (it does locally) when being the last one.

Reply:
@rostislav-deriv wouldn't this typically mean that there's a leaking state or lack of cleanup? Could this be fixed by adding a `jest.clearAllMocks()` in the beforeEach/afterEach block?

Reply:
Right @lubega-deriv. This might be due to some mocks and as suggested can be tested by `clearAllMocks`

Reply:
Have tried doing this in https://github.com/binary-com/deriv-app/pull/15186/commits/7a50d017d903dc97d6eac4dc1259dbd8d10c875a, not working
i believe it's caused by the changes being made to `window`

Reply:
Fixed in https://github.com/binary-com/deriv-app/pull/15186/commits/d81866657aa75f8e6c1bbfc74601186b82820f98